### PR TITLE
Add country-related endpoints documentation

### LINKED ISSUE #114: Add documentation regarding the upcoming country-related endpoints

### PR DESCRIPTION
**Pull request by ruiquelhas**
Resolves #114.

### CODE REVIEW

File: _countries.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+  "currency": "USD",
+  "name": "United States Of America",
+  "phone": {
+    "code": "1",
+    "example": "(201) 555-5555"
+  }
+}
+```
+
+Retrieve the information about a single country (provided its `alpha2` code).
```
Comment (rplopes) [inline _countries.md]: "Retrieves" for consistency.

File: _countries.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+}, {
+  "code": "US-WV",
+  "name": "West Virginia"
+}, {
+  "code": "US-WY",
+  "name": "Wyoming"
+}]
+```
+
+Retrieve the list of subdivisions of a given country (provided its `alpha2` code).
```
Comment (rplopes) [inline _countries.md]: "Retrieves" for consistency.

**Comment by rplopes** [comment]
Other than my small comments, LGTM.

**Comment by ricardobcl** [comment]
This is deprecated, closing.

**Pull request closed without merging**